lib/gds_api/test_helpers/publishing_api_v2.rb

```diff
@@ -296,7 +296,7 @@ def publishing_api_has_content(items, params = {})
       # publishing_api_has_content allows for flexible passing in of arguments, please use instead
       def publishing_api_has_fields_for_document(document_type, items, fields)
         body = Array(items).map { |item|
-          item.with_indifferent_access.slice(*fields)
+          deep_stringify_keys(item, &:to_sym).slice(*fields)
```

Comment:
Good spot - I think it could be either `deep_transform_keys` or `deep_stringify_keys`, as the body will be converted to json on line 308, but if I'm going to use `deep_stringify_keys` I shouldn't pass in a block. Amended in 2157c77.